pkg/client/cli/cmd_status.go

```diff
@@ -250,9 +248,13 @@ func (cs *userDaemonStatus) WriteTo(out io.Writer) (int64, error) {
 		n += ioutil.Printf(out, "  Install ID        : %s\n", cs.InstallID)
 		if tms := cs.TrafficManagerStatus; tms != nil {
 			n += ioutil.Println(out, "Traffic Manager: Running")
```

Comment:
"Running/Not connected" doesn't add up. It should be "Connected/Not connected" or "Running/Not running". Since we cannot determine if it isn't running, unless we're connected, we must use the former.